datafusion/core/tests/sqllogictests/README.md

```diff
@@ -97,17 +97,27 @@ Query records follow the format:
 
 ```sql
 # <test_name>
-query <type_string> <sort_mode> <label>
+query <type_string> <sort_mode>
 <sql_query>
 ----
 <expected_result>
 ```
 
 - `test_name`: Uniquely identify the test name (arrow-datafusion only)
-- `type_string`: A short string that specifies the number of result columns and the expected datatype of each result column. There is one character in the <type_string> for each result column. The characters codes are "T" for a text result, "I" for an integer result, and "R" for a floating-point result.
-- (Optional) `label`: sqllogictest stores a hash of the results of this query under the given label. If the label is reused, then sqllogictest verifies that the results are the same. This can be used to verify that two or more queries in the same test script that are logically equivalent always generate the same output.
```

Comment:
I removed the `label` from README for now, because the label semantics are not really supported in `sqllogictest-rs` for now https://github.com/risinglightdb/sqllogictest-rs/blob/a49cc0004627f43467a9dbafef500e96cb0b8150/sqllogictest/src/runner.rs#L498-L499.